ic2datadog.py

```diff
@@ -63,16 +63,18 @@
 
         for metric in node["payload"]:
             dd_metric_name = 'instaclustr.{0}'.format(metric["metric"])
-            if metric["metric"] == "nodeStatus":
-                # node status metric maps to a data dog service check
-                if metric["values"][0]["value"] =="WARN":
-                    statsd.service_check(dd_metric_name, 1, tags=configuration['tags'] + tag_list) # WARN status
+            if len(metric["values"]) > 0:
+                if metric["metric"] == "nodeStatus":
+                    # node status metric maps to a data dog service check
+                    if metric["values"][0]["value"] =="WARN":
```

Comment:
The workaround should work for https://instaclustr.zendesk.com/agent/tickets/206250.
However, the problem might be that this line assumes the `metric["values"][0]["value"]` exists in the right format (maybe need to validate here?). Since we control data format, I probably not worry too much here.